lib/experimental/fonts/font-library/font-library.php

```diff
@@ -32,9 +32,45 @@ function gutenberg_init_font_library_routes() {
 	);
 	register_post_type( 'wp_font_family', $args );
 
+	register_post_type(
+		'wp_font_face',
+		array(
+			'labels'       => array(
+				'name'          => __( 'Font Faces', 'gutenberg' ),
+				'singular_name' => __( 'Font Face', 'gutenberg' ),
+			),
+			'public'       => false,
+			'_builtin'     => true,                              /* internal use only. don't use this when registering your own post type. */
+			'hierarchical' => false,
+			'show_in_rest' => false,
+			'rest_base'    => 'font-faces',
```

Comment:
`'rest_controller_class' => "WP_REST_Font_Faces_Controller"`

This should have been used here. Not manaul registeration